src/script/app/eventemitter.ts

```diff
@@ -0,0 +1,8 @@
+import mitt from 'mitt';
+import type {QualifiedId} from '@wireapp/api-client/src/user/';
+
+type Events = {
+  'conversation.delete': QualifiedId;
```

Comment:
We probably do not need a huge object to define all the possible events, we can stay with string identifiers since now TS checks them. Open for discussion